src/utils/filter-videos.js

```diff
@@ -0,0 +1,8 @@
+export const getFilteredItems = (items, filter) =>
+  items.filter(({ id }) => {
+    if (`youtube#${filter}` === id.kind) {
+      return true;
+    }
+
+    return false;
+  });
```

Comment:
could be simplified to:
```suggestion
export const getFilteredItems = (items, filter) =>
  items.filter(({ id }) => `youtube#${filter}` === id.kind);
```